packit/config.py

```diff
@@ -130,6 +131,7 @@ def get_from_dict(cls, raw_dict: dict, validate=True) -> "Config":
         config.github_app_id = raw_dict.get("github_app_id", "")
         config.github_app_cert_path = raw_dict.get("github_app_cert_path", "")
         config.webhook_secret = raw_dict.get("webhook_secret", "")
+        config.actions_environment = raw_dict.get("actions_handler", "local")
```

Comment:
would be nice to be consistent here:
```python
config.actions_handler = raw_dict.get("actions_handler", "local")
```

Reply:
When I wrote it, I did not catch it, that variables equal values in packit.yaml file. Upss.
Good catch.